python/edgar_filing_searcher/parsers/parser_utils.py

```diff
@@ -0,0 +1,100 @@
+"""This file contains functions used in main.py"""
+import logging
+
+from edgar_filing_searcher.database import db
+from edgar_filing_searcher.models import Company, EdgarFiling, Data13f
+from edgar_filing_searcher.parsers.daily_index_crawler import \
+    ensure_13f_filing_detail_urls, get_subdirectories_for_specific_date
+from edgar_filing_searcher.parsers.parser_class import Parser
+from edgar_filing_searcher.errors import BadWebPageResponseException, NoUrlException, \
+    NoAccessionNumberException, InvalidUrlException
+
+
+def create_url_list(date_):
+    """This function creates a list of 13F URLs"""
+    subdirectories = get_subdirectories_for_specific_date(date_)
+    if not subdirectories:
+        logging.info('No cik_no_and_accession_nos for date: %s', date_)
```

Comment:
I suggest making this normal text. `"No CIK number and accession numbers for ..."`